cppframework/Foo.cpp

```diff
@@ -2,9 +2,6 @@
 #include <string>
 #include <iostream>
 
-#define EXPORT __attribute__((visibility("default")))
```

Comment:
Without these lines, the C++ framework has a symbol visibility error. This causes the app's usage of the framework to create a compiler error.

<img width="1175" alt="screen shot 2017-07-06 at 11 11 32 pm" src="https://user-images.githubusercontent.com/733784/27941801-7fc0d6fc-62a0-11e7-9ed8-ab3322b1266a.png">